sdk/search/search-documents/README.md

```diff
@@ -811,7 +860,24 @@ async function main() {
 main();
 ```
 
-### Get the status of an indexer
+#### Retrieve an existing datasource and modify a field in it
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
+
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+async function main() {
+  let dataSource = await client.getDataSource('my-data-source-2');
+  console.log(`Container Name: ${dataSource.container.name}`);
+  dataSource.container.name = 'my-container-2';
```

Comment:
according to [the docs](https://docs.microsoft.com/en-us/rest/api/searchservice/create-data-source) the container name "specifies the SQL API collection" does this have to be a particular name for the data source to work to import from the sample credentials?

Reply:
Nope. It does not do that validation. Any name works